database/seeders/DatabaseSeeder.php

```diff
@@ -16,8 +16,9 @@ public function run(): void
         // User::factory(10)->create();
 
         User::factory()->create([
-            'name' => 'Test User',
-            'email' => 'test@example.com',
+            'name' => 'admin',
+            'email' => 'admin@example.com',
+            'password' => 'password'
```

Comment:
plain text passwords can be `bruteforce` easily. please encrypt the password using `bcrypt` mechanism

Reply:
okay sure got it, will use bcrypt #7